src/common/FilePreviewCommon/Assets/Monaco/index.html

```diff
@@ -63,15 +63,26 @@
         require.config({ paths: { vs: 'http://[[PT_URL]]/monacoSRC/min/vs' } });
         require(['vs/editor/editor.main'], async function () {
             await registerAdditionalLanguages(monaco)
+
+            // Creates a theme to handle custom tokens
+            monaco.editor.defineTheme('theme', {
+                base: theme, // Sets the base theme to "vs" or "vs-dark" depending on the user's preference
+                inherit: true,
+                rules: [
+                    {token: 'custom-gitignore.negation', foreground: '00f7e5'}
```

Comment:
Wonder if we can have a separate `customTokenColors` in the assets folder near by the custom languages? 🤔

Reply:
Feels that such an extra file is easier to find if required for other new tokenizers.